docs/UserGuide.md

```diff
@@ -66,13 +66,17 @@ Adds an event to the event list, a participant to an event, or an item to an eve
 
 Format:  
 
-* `add -e EVENT -t TIME -v VENUE` for adding an event to the events list.
+* `add -e EVENT -t TIME -v VENUE -u PRIORITY` for adding an event to the events list.
 * `add -p PARTICIPANT -n NUMBER -email EMAIL -e EVENT` for adding a participant to an event.
 * `add -m ITEM -e EVENT` for adding an item to an event.
 
+Remarks:
+* `TIME` must be entered in the format `yyyy-mm-dd HH:mm`.
+* `PRIORITY` must be either `HIGH`, `MEDIUM`, or `LOW`.
```

Comment:
user input for priority is not case-sensitive

Reply:
Noted, will amend accordingly.